single-user-d4science/Dockerfile

```diff
@@ -47,7 +52,8 @@ RUN pip install shortid \
         geopy==2.0.0 \
         hvplot==0.6.0 \
         intake==0.6.0 \
-        intake-esm==2020.8.15
+        intake-esm==2020.8.15 \
+        GDAL==3.0.4 #need to match the version of gdal installed
```

Comment:
maybe then pin the gdal version above too? so no issues appear if the apt side of things gets updated?

Reply:
pinned!